jdaviz/configs/default/plugins/model_fitting/model_fitting.vue

```diff
@@ -212,6 +212,9 @@
     methods: {
       componentInEquation(componentId) {
         return this.model_equation.split(/[+*\/-]/).indexOf(componentId) !== -1
+      },
+      roundUncertainty(uncertainty) {
+        return uncertainty.toPrecision(2)
```

Comment:
I think this is probably fine for now, but we might need to make it more advanced for some edge cases with really large/small uncertainties (or if we ever display values in scientific notation).

Reply:
Oooo I wish I knew this trick for the Angle PR. 😆